### DISCUSSION TITLE
Custom system requirements?

### DISCUSSION THREAD

**Discussion by hgraca** [composer/composer#11396] [Q&A / Support] [answered]
Hi, 

I'm wondering if there is a way to easily implement custom system requirements.

For example, let's say my project needs bash ^5.1, I could create a plugin that would understand the requirement 'bash' and would know how to check it's version.

Is something like this possible?

If so, how?

Tkx a lot in advance :)

**Comment by Seldaek** [comment, marked as answer]
Nope, nope, that sounds impossible to me sorry :)

**Reply by hgraca** [reply]
Ok, tkx for taking some time with this anyway. :)
